polaris-ui/src/app/common/presentation/layout/Layout.tsx

```diff
@@ -27,35 +28,46 @@ export const Layout: React.FC<LayoutProps> = ({ isWide, children }) => {
       (skipLinkSiblingRef.current as HTMLButtonElement).blur();
     }
   }, [location.pathname]);
+  const { globalNav } = useUserGroupsFeatureFlag();
 
   return (
     <>
       <div ref={skipLinkSiblingRef} tabIndex={-1} />
-      <SkipLink href="#main-content">Skip to main content</SkipLink>
-      <header
-        className="govuk-header "
-        role="banner"
-        data-module="govuk-header"
-      >
-        <div className={`govuk-header__container`}>
-          <div className={containerCssClass}>
-            <div className={`govuk-header__logo ${classes.logo} `}>
-              <a
-                href="/"
-                className="govuk-header__link govuk-header__link--homepage"
-                data-testid="link-homepage"
-              >
-                <span className="govuk-header__logotype">
-                  <span className="govuk-header__logotype-text">
-                    Crown Prosecution Service
+      {globalNav ? (
+        <div className={containerCssClass}>
```

Comment:
Optimally there would only be one div with `className={containerCssClass}`.  The existing code (which I wrote 🙂 ) has one within the header and one around the content.  To keep the PR as simple as possible I've avoided refactoring and just given the global header its own container (the global header expands to fill its container - it has no inherit width specified)